### PR TITLE
add pascals-triangle

### PR DESCRIPTION
**Pull request by Average-user**

### CODE REVIEW

**Comment by Average-user** [comment]
I don't understand the Travis failure, but apparently has to do with ```#lang lazy``` I think.

**Comment by benreyn** [comment]
@Average-user Thanks for the PR here. I was recently added as a maintainer for most of the lisp-flavored languages. Ill take a look at this and #91 some time this weekend.

File: exercises/pascals-triangle/example.rkt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#lang lazy
```
Comment (benreyn) [inline exercises/pascals-triangle/example.rkt]: I think your right about why the tests are failing on CI. I think we should stick to `#lang racket`.

**Comment by zenspider** [comment]
We test using the racket-minimal package. `lazy` is probably not distributed with it? Not sure how to resolve (tho lazy should be easy to implement on your side if need be).

**Pull request closed without merging**